### PR TITLE
removed flask and django upper bound

### PR DESCRIPTION
**Pull request by g-as**
As discussed on discord, upper bound relaxing on flask and django.

Should I add tests for flask 1.1.x?

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/), so make sure to pick the appropriate type. If in doubt feel free to ask :)


Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to Guillaume Andreu Sabater for the PR 👏

Get it here 👉 https://github.com/strawberry-graphql/strawberry/releases/tag/(next)
```

**Comment by DoctorJohn** [comment]
Is Flask 1.x still supported by the Flask maintainers? If so we could create a test matrix (like the one we have in place for officially supported Django versions). I had no luck finding out whether Flask supports any other version than their latest one.

**Comment by g-as** [comment]
Same here, I looked for a support policy for 1.1.x to no avail.

**Review by patrick91 — APPROVED** [review]
Thanks!

File: RELEASE.md
Comment (patrick91) [inline RELEASE.md:3]:
```suggestion
This release adds support for flask 2.x and also relaxes the requirements for Django, allowing to install newer version of Django without having to wait for Strawberry to update its supported dependencies list.
```

**Pull request merged**
